### ISSUE TITLE
Installation errors

### ISSUE THREAD

**Issue by skeeler** [microsoft/Microsoft365DSC#592]
#### Details of the scenario you tried and the problem that is occurring
Attempting to install the Microsoft365DSC module following the instructions at: http://microsoft365dsc.com/Pages/GettingStarted/#Install

#### Verbose logs showing the problem
PS C:\WINDOWS\system32> Install-Module Microsoft365DSC -Force
PackageManagement\Install-Package : The following commands are already available on this system:'Get-AzureADApplicationProxyConnectorGroupMembers,Add-AzureADApplicationOwner,Add-AzureADDeviceR
egisteredOwner,Add-AzureADDeviceRegisteredUser,Add-AzureADDirectoryRoleMember,Add-AzureADGroupMember,Add-AzureADGroupOwner,Add-AzureADMSApplicationOwner,Add-AzureADMSLifecyclePolicyGroup,Add-A
zureADServicePrincipalOwner,Confirm-AzureADDomain,Connect-AzureAD,Disconnect-AzureAD,Enable-AzureADDirectoryRole,Get-AzureADApplication,Get-AzureADApplicationExtensionProperty,Get-AzureADAppli
cationKeyCredential,Get-AzureADApplicationLogo,Get-AzureADApplicationOwner,Get-AzureADApplicationPasswordCredential,Get-AzureADApplicationProxyApplication,Get-AzureADApplicationProxyApplicatio
nConnectorGroup,Get-AzureADApplicationProxyConnector,Get-AzureADApplicationProxyConnectorGroup,Get-AzureADApplicationProxyConnectorMemberOf,Get-AzureADApplicationServiceEndpoint,Get-AzureADCon
tact,Get-AzureADContactDirectReport,Get-AzureADContactManager,Get-AzureADContactMembership,Get-AzureADContactThumbnailPhoto,Get-AzureADContract,Get-AzureADCurrentSessionInfo,Get-AzureADDeleted
Application,Get-AzureADDevice,Get-AzureADDeviceConfiguration,Get-AzureADDeviceRegisteredOwner,Get-AzureADDeviceRegisteredUser,Get-AzureADDirectoryRole,Get-AzureADDirectoryRoleMember,Get-AzureA
DDirectoryRoleTemplate,Get-AzureADDomain,Get-AzureADDomainNameReference,Get-AzureADDomainServiceConfigurationRecord,Get-AzureADDomainVerificationDnsRecord,Get-AzureADExtensionProperty,Get-Azur
eADGroup,Get-AzureADGroupAppRoleAssignment,Get-AzureADGroupMember,Get-AzureADGroupOwner,Get-AzureADMSApplication,Get-AzureADMSApplicationExtensionProperty,Get-AzureADMSApplicationOwner,Get-Azu
reADMSDeletedDirectoryObject,Get-AzureADMSDeletedGroup,Get-AzureADMSGroup,Get-AzureADMSGroupLifecyclePolicy,Get-AzureADMSIdentityProvider,Get-AzureADMSLifecyclePolicyGroup,Get-AzureADOAuth2Per
missionGrant,Get-AzureADObjectByObjectId,Get-AzureADServiceAppRoleAssignedTo,Get-AzureADServiceAppRoleAssignment,Get-AzureADServicePrincipal,Get-AzureADServicePrincipalCreatedObject,Get-AzureA
DServicePrincipalKeyCredential,Get-AzureADServicePrincipalMembership,Get-AzureADServicePrincipalOAuth2PermissionGrant,Get-AzureADServicePrincipalOwnedObject,Get-AzureADServicePrincipalOwner,Ge
t-AzureADServicePrincipalPasswordCredential,Get-AzureADSubscribedSku,Get-AzureADTenantDetail,Get-AzureADTrustedCertificateAuthority,Get-AzureADUser,Get-AzureADUserAppRoleAssignment,Get-AzureAD
UserCreatedObject,Get-AzureADUserDirectReport,Get-AzureADUserExtension,Get-AzureADUserLicenseDetail,Get-AzureADUserManager,Get-AzureADUserMembership,Get-AzureADUserOAuth2PermissionGrant,Get-Az
ureADUserOwnedDevice,Get-AzureADUserOwnedObject,Get-AzureADUserRegisteredDevice,Get-AzureADUserThumbnailPhoto,Get-CrossCloudVerificationCode,New-AzureADApplication,New-AzureADApplicationExtens
ionProperty,New-AzureADApplicationKeyCredential,New-AzureADApplicationPasswordCredential,New-AzureADApplicationProxyApplication,New-AzureADApplicationProxyConnectorGroup,New-AzureADDevice,New-
AzureADDomain,New-AzureADGroup,New-AzureADGroupAppRoleAssignment,New-AzureADMSApplication,New-AzureADMSApplicationExtensionProperty,New-AzureADMSApplicationKey,New-AzureADMSApplicationPassword
,New-AzureADMSGroup,New-AzureADMSGroupLifecyclePolicy,New-AzureADMSIdentityProvider,New-AzureADMSInvitation,New-AzureADServiceAppRoleAssignment,New-AzureADServicePrincipal,New-AzureADServicePr
incipalKeyCredential,New-AzureADServicePrincipalPasswordCredential,New-AzureADTrustedCertificateAuthority,New-AzureADUser,New-AzureADUserAppRoleAssignment,Remove-AzureADApplication,Remove-Azur
eADApplicationExtensionProperty,Remove-AzureADApplicationKeyCredential,Remove-AzureADApplicationOwner,Remove-AzureADApplicationPasswordCredential,Remove-AzureADApplicationProxyApplication,Remo
ve-AzureADApplicationProxyApplicationConnectorGroup,Remove-AzureADApplicationProxyConnectorGroup,Remove-AzureADContact,Remove-AzureADContactManager,Remove-AzureADDeletedApplication,Remove-Azur
eADDevice,Remove-AzureADDeviceRegisteredOwner,Remove-AzureADDeviceRegisteredUser,Remove-AzureADDirectoryRoleMember,Remove-AzureADDomain,Remove-AzureADGroup,Remove-AzureADGroupAppRoleAssignment
,Remove-AzureADGroupMember,Remove-AzureADGroupOwner,Remove-AzureADMSApplication,Remove-AzureADMSApplicationExtensionProperty,Remove-AzureADMSApplicationKey,Remove-AzureADMSApplicationOwner,Rem
ove-AzureADMSApplicationPassword,Remove-AzureADMSDeletedDirectoryObject,Remove-AzureADMSGroup,Remove-AzureADMSGroupLifecyclePolicy,Remove-AzureADMSIdentityProvider,Remove-AzureADMSLifecyclePol
icyGroup,Remove-AzureADOAuth2PermissionGrant,Remove-AzureADServiceAppRoleAssignment,Remove-AzureADServicePrincipal,Remove-AzureADServicePrincipalKeyCredential,Remove-AzureADServicePrincipalOwn
er,Remove-AzureADServicePrincipalPasswordCredential,Remove-AzureADTrustedCertificateAuthority,Remove-AzureADUser,Remove-AzureADUserAppRoleAssignment,Remove-AzureADUserExtension,Remove-AzureADU
serManager,Reset-AzureADMSLifeCycleGroup,Restore-AzureADDeletedApplication,Restore-AzureADMSDeletedDirectoryObject,Revoke-AzureADSignedInUserAllRefreshToken,Revoke-AzureADUserAllRefreshToken,S
elect-AzureADGroupIdsContactIsMemberOf,Select-AzureADGroupIdsGroupIsMemberOf,Select-AzureADGroupIdsServicePrincipalIsMemberOf,Select-AzureADGroupIdsUserIsMemberOf,Set-AzureADApplication,Set-Az
ureADApplicationLogo,Set-AzureADApplicationProxyApplication,Set-AzureADApplicationProxyApplicationConnectorGroup,Set-AzureADApplicationProxyApplicationCustomDomainCertificate,Set-AzureADApplic
ationProxyApplicationSingleSignOn,Set-AzureADApplicationProxyConnector,Set-AzureADApplicationProxyConnectorGroup,Set-AzureADDevice,Set-AzureADDomain,Set-AzureADGroup,Set-AzureADMSApplication,S
et-AzureADMSApplicationLogo,Set-AzureADMSGroup,Set-AzureADMSGroupLifecyclePolicy,Set-AzureADMSIdentityProvider,Set-AzureADServicePrincipal,Set-AzureADTenantDetail,Set-AzureADTrustedCertificate
Authority,Set-AzureADUser,Set-AzureADUserExtension,Set-AzureADUserLicense,Set-AzureADUserManager,Set-AzureADUserPassword,Set-AzureADUserThumbnailPhoto,Update-AzureADSignedInUserPassword'.
This module 'AzureADPreview' may override the existing commands. If you still want to install this module 'AzureADPreview', use -AllowClobber parameter.
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\2.2.1\PSModule.psm1:9683 char:34
+ ... talledPackages = PackageManagement\Install-Package @PSBoundParameters
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], Exception
    + FullyQualifiedErrorId : CommandAlreadyAvailable,Validate-ModuleCommandAlreadyAvailable,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# insert configuration here
```

#### The operating system the target node is running
OsName               : Microsoft Windows 10 Enterprise
OsOperatingSystemSKU : EnterpriseEdition
OsArchitecture       : 64-bit
WindowsVersion       : 2004
WindowsBuildLabEx    : 19041.1.amd64fre.vb_release.191206-1406
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version of the DSC module that was used ('dev' if using current dev branch)
1.0.5.128 (current)

**Comment by NikCharlebois** [comment]
Hi @skeeler that is most likely because you also have the AzureAD module installed already. Try running:

Install-Module Microsoft365DSC -AllowClobber -Force

**Comment by skeeler** [comment]
I've got AzureAD module version 2.0.2.76 (latest) installed - will using -AllowClobber preserve latest version or does the Microsoft365DSC module reference an earlier version?

**Comment by NikCharlebois** [comment]
It references the AzureADPreview module, which leverages the Graph

**Comment by NikCharlebois** [comment]
@skeeler is your issue resolved?

**Comment by skeeler** [comment]
Hi @NikCharlebois , thanks for following up. I tried uninstalling AzureAD to make way for AzureADPreview but got blocked because it was in use. I'm a little leery of using the preview (in case it causes issues with other projects/work I have using AzureAD. I checked the feature changes between prod/preview and they appear to be MSAppManagement related. Does M365DSC actually need the AzureADPreview or could it target the latest production version?
